src/edu/washington/escience/myria/CsvTupleWriter.java

```diff
@@ -58,6 +61,11 @@ private CsvTupleWriter(final OutputStream out, final CSVFormat csvFormat) throws
   }
 
   @Override
+  public void open(final OutputStream stream) throws IOException {
```

Comment:
I favor removing the old public constructors and upgrading all callers to use the new `open()` method. In general, I think we should avoid non-serializable parameters in operator-related constructors. @jingjingwang what do you think?